web/src/test/java/com/bazaarvoice/emodb/web/config/DatadogMetricFilterTest.java

```diff
@@ -127,10 +126,10 @@ public void testExpansionFilterExclusion() throws Exception {
         reporter.report();
 
         // Verify only the desired metrics were sent.  Notably min, max, and the nth percentiles should be absent.
-        verify(_request).addCounter(argThat(hasCounter("test.histogram.count", 3)));
+        verify(_request).addGauge(argThat(hasGauge("test.histogram.count", 3)));
         verify(_request).addGauge(argThat(hasGauge("test.histogram.mean", 2)));
         verify(_request).addGauge(argThat(hasGauge("test.histogram.median", 2)));
-        verify(_request).addGauge(argThat(hasGauge("test.histogram.stddev", 1.0)));
+        verify(_request).addGauge(argThat(hasGauge("test.histogram.stddev", 0.816496580927726)));
```

Comment:
any impact on our monitors/alerts?

Reply:
Dropwizard metrics changed standard deviation to be population standard deviation, rather than sample standard deviation. As `n` grows, the output of these should converge. Additionally, I don't think we have any metrics related to standard deviation.